static/js/containers/admin/ChannelModerationPage.js

```diff
@@ -133,13 +145,21 @@ export class ChannelModerationPage extends React.Component<Props> {
         <EditChannelNavbar channelName={channel.name} />
 
         <div className="channel-moderation">
-          {reports.length === 0 ? (
-            <Card title="Reported Posts & Comments">
-              <div className="empty-message">No outstanding reports</div>
-            </Card>
-          ) : (
-            reports.map(this.renderReport)
-          )}
+          {withLoadingRender({
+            errored,
+            loaded,
+            notAuthorized,
+            notFound,
+            LoadingComponent: Loading,
+            render:           () =>
+              reports.length === 0 ? (
+                <Card title="Reported Posts & Comments">
+                  <div className="empty-message">No outstanding reports</div>
+                </Card>
+              ) : (
+                reports.map(this.renderReport)
+              )
+          })}
```

Comment:
it might make sense to pull this anonymous function into a `renderReports` instance method for readability